nvflare/apis/server_engine_spec.py

```diff
@@ -115,3 +115,15 @@ def persist_components(self, fl_ctx: FLContext, completed: bool):
 
     def restore_components(self, snapshot: RunSnapshot, fl_ctx: FLContext):
         pass
+
+    def send_admin_requests(self, requests):
+        pass
```

Comment:
note that if this is specified here, we are requiring the ServerEngine implementation to always have an admin.